docs/production-deployment/cloud/account-setup/users.mdx

```diff
@@ -259,59 +272,59 @@ Account Global Admin has Namespace Admin permissions on Namespaces.
+| GetAccountSettings                    | ✔                  | ✔         | ✔             | ✔            | ✔             |
+| GetAccountUsage                       |                    |           |               | ✔            | ✔             |
+| GetAPIKey                             | ✔                  | ✔         | ✔             | ✔            | ✔             |
+| GetAPIKeys                            | ✔                  | ✔         | ✔             | ✔            | ✔             |
+| GetAsyncOperation                     | ✔                  | ✔         | ✔             | ✔            | ✔             |
+| GetDecodedCertificate                 | ✔                  | ✔         | ✔             | ✔            | ✔             |
+| GetIdentities                         | ✔                  | ✔         | ✔             | ✔            | ✔             |
+| GetIdentity                           | ✔                  | ✔         | ✔             | ✔            | ✔             |
+| GetNamespaces                         | ✔                  | ✔         | ✔             | ✔            | ✔             |
+| GetNamespacesUsage                    |                    |           |               | ✔            | ✔             |
+| GetRegion                             | ✔                  | ✔         | ✔             | ✔            | ✔             |
+| GetRegions                            | ✔                  | ✔         | ✔             | ✔            | ✔             |
+| GetRequestStatus                      | ✔                  | ✔         | ✔             | ✔            | ✔             |
+| GetRequestStatuses                    |                    |           |               | ✔            | ✔             |
+| GetRequestStatusesForNamespace        | ✔                  | ✔         | ✔             | ✔            | ✔             |
+| GetRequestStatusesForUser             | ✔                  | ✔         | ✔             | ✔            | ✔             |
+| GetRoles                              | ✔                  | ✔         | ✔             | ✔            | ✔             |
+| GetRolesByPermissions                 | ✔                  | ✔         | ✔             | ✔            | ✔             |
+| GetServiceAccount                     | ✔                  | ✔         | ✔             | ✔            | ✔             |
+| GetServiceAccounts                    | ✔                  | ✔         | ✔             | ✔            | ✔             |
+| GetStripeInvoice                      |                    |           | ✔             |              | ✔             |
+| GetUser                               | ✔                  | ✔         | ✔             | ✔            | ✔             |
+| GetUsers                              | ✔                  | ✔         | ✔             | ✔            | ✔             |
+| GetUsersWithAccountRoles              | ✔                  | ✔         | ✔             | ✔            | ✔             |
```

Comment:
These apis are technically part of Read-Only role today but we are in the process of migrating to Admin role--we should leave this alone for now and not mention that in the docs.